timApp/plugin/pluginControl.py

```diff
@@ -458,6 +458,11 @@ def pluginify(
     taketime("answ", "start")
     if not view_ctx.preview and has_edit_access(doc.get_docinfo()):
         for p in pars:
+            # Do not add the 'Translation out of date' and 'Check translation' markings to
+            # area_end pars if they are empty.
+            p_html_cache = p.html_cache.get(p.attrs.get("rt")) if p.html_cache else None
+            if p.get_attr("area_end") and not p.get_markdown() and not p_html_cache:
+                continue
```

Comment:
Toimisiko tämä koko PR jos sama tarkistus ja skippaus tehtäisiin myös arean alkupäähän?

Tein tuollaisen dokumentin: https://timbeta02.tim.education/view/users/test-user-1/test-area-tl/en-GB

Tein niin, että ekaksi tein suomenkielisen version, sitten tein käännöksen ja sitten laitoin suomenkielisesä versiossa kaksi lohkoa alueen sisälle. Silloin kuitenkin alkuun jää tuo turhahko Translation out of date alueen alusta.
Jos tämä sama skippaus tehtäisiin myös alueen alkuun, niin se ehkä näyttäisi heti alkuun OKlta. Ehkä silloin ei myöskään tarvittaisi tehdä muita säätöjä kuin vain tähän paikkaan?